docs/Build-for-RPi2.md

```diff
@@ -3,10 +3,11 @@
 ### Prerequisite
 #### Linux
 
-Install arm linux cross compiler.
+Install arm linux cross compiler and i2c library.
 
 ``` bash
-sudo apt-get install gcc-arm-linux-gnueabihf g++-arm-linux-gnueabihf
+sudo apt-get install gcc-arm-linux-gnueabihf g++-arm-linux-gnueabihf i2c-tools libi2c-dev
+sudo mv /usr/arm-linux-gnueabihf/include/linux/i2c-dev.h /usr/arm-linux-gnueabihf/include/linux/i2c-dev.h.kernel
```

Comment:
Sounds good to me. (Note, I'm not that familiar with I2C implementations. All my above comments were along the lines of what's favourable from build system or user perspective.) The licensing of the linked code seems to fit the purposes of IoT.js, as far as I can tell.